### DISCUSSION TITLE
为APP添加指纹解锁功能(Add fingerprint unlock to app)

### DISCUSSION THREAD

**Discussion by cutelolly** [community/community#50542] [Mobile]
为了保证一定的安全性，可以为github APP添加指纹解锁功能。

**Comment by aconsuegra** [comment]
Thanks for your feature request @cutelolly! I'm going to close this one as duplicated of https://github.com/orgs/community/discussions/3842, please feel free to upvote it!

**Reply by Rianbautista** [reply]
> Thanks for your feature request @cutelolly! I'm going to close this one as duplicated of https://github.com/orgs/community/discussions/3842, please feel free to upvote it!

**Comment by Rianbautista** [comment]
> Thanks for your feature request @cutelolly! I'm going to close this one as duplicated of https://github.com/orgs/community/discussions/3842, please feel free to upvote it!

**Comment by Rianbautista** [comment]
> 为了保证一定的安全性，可以为github APP添加指纹解锁功能。
